lib/modules/datasource/conda/prefix-dev.ts

```diff
@@ -0,0 +1,159 @@
+import is from '@sindresorhus/is';
+import { z } from 'zod';
+import { logger } from '../../../logger';
+import { isNotNullOrUndefined } from '../../../util/array';
+import type { Http } from '../../../util/http';
+import type { Timestamp } from '../../../util/timestamp';
+import { MaybeTimestamp } from '../../../util/timestamp';
+import type { ReleaseResult } from '../types';
+
+const MAX_PREFIX_DEV_GRAPHQL_PAGE = 10;
+
+const File = z.object({
+  version: z.string(),
+  createdAt: z.string().nullable(),
+  yankedReason: z.string().nullable(),
+});
+
+const Version = z.object({
+  version: z.string(),
+});
+
+const PagedResponseSchema = z.object({
+  data: z.object({
+    data: z.object({
+      data: z
+        .object({
+          pages: z.number(),
+          page: z.array(z.unknown()),
+        })
+        .nullable(),
+    }),
+  }),
+});
+
+export async function getReleases(
+  http: Http,
+  channel: string,
+  packageName: string,
+): Promise<ReleaseResult | null> {
+  logger.debug(
+    { channel, packageName },
+    'lookup package from prefix.dev graphql API',
+  );
+
+  const versions = await getPagedResponse(
```

Comment:
prettier will be possible to format graphql if we have `gql` tag here, but graphql-tag is currently not in our dep tree so I didn't add it.

Reply:
but `codeBlock ` is a dev dependency